### ISSUE TITLE
set up subdomain for whw 2023 website

### ISSUE THREAD

**Issue by aaarendt** [waterhackweek/website2023#1]
@scottyhq, @jomey can you add "water" as a hackweek.io subdomain so I can get an initial splash page up for WHW?

**Comment by jomey** [comment]
water.hackweek.io is now pointing to waterhackweek.github.io

Is there a need for an archival URL for the past event website?

**Comment by aaarendt** [comment]
Thanks @jomey! I looked and previous year's websites are not archived so I don't think there's a need for previous years.

So far I copied our website template over to this organization and called it website2023. Then I was able to give it the custom domain water.hackweek.io.

I'm not sure how that meshes with you pointing the subdomin to a specific url? Let me know if I'm following the workflow on this that we set up.
